src/lang/uk.rs

```diff
@@ -0,0 +1,1233 @@
+     "дванадцят",  
+     "тринадцят",  
+     "чотирнадцят",
+     "пʼятнадцят", 
+     "шістнадцят", 
+     "сімнадцят",  
+     "вісімнадцят",
+     "девʼятнадцят"
+];
+
+const TEENS_FLEXIONS: [&str; 6] = [
+     "ь", "и", "и", "ь", "ьма", "и"        
+];
+
+#[rustfmt::skip]
+const TENS: [[&str; 6]; 8] = [
+    [ "двадцять",   "двадцяти",    "двадцяти",    "двадцять",   "двадцятьма",     "двадцяти"    ],
+    [ "тридцять",   "тридцяти",    "тридцяти",    "тридцять",   "тридцятьма",    "тридцяти"    ],
+    [ "сорок",      "сорока",      "сорока",      "сорок",      "сорока",        "сорока"      ],
+    [ "пʼятдесят",  "пʼятдесяти",  "пʼятдесяти",  "пʼятдесят",  "пʼятдесятьма",  "пʼятдесяти"  ],
+    [ "шістдесят",  "шістдесяти",  "шістдесяти",  "шістдесят",  "шістдесятьма",  "шістдесяти"  ],
+    [ "сімдесят",   "сімдесяти",   "сімдесяти",   "сімдесять",  "сімдесятьма",   "сімдесяти"   ],
+    [ "вісімдесят", "вісімдесяти", "вісімдесяти", "вісімдесят", "вісімдесятьма", "вісімдесяти" ],
+    [ "девʼяносто", "девʼяноста",  "девʼяноста",  "девʼяносто", "девʼяноста",    "девʼяноста"  ],
+];
+
+const ORDINAL_TENS_BASES: [&str; 9] = [
+    "десят", "двадцят", "тридцят", "сороков", "пʼятдесят", "шістдесят", "сімдесят", "вісімдесят", "девʼяност"
+];
+
+#[rustfmt::skip]
+const HUNDREDS: [[&str; 6]; 9] = [
+    [ "сто",       "ста",         "ста",          "сто",       "ста",            "ста"          ],
+    [ "двісті",    "двохсот",     "двомстам",     "двісті",    "двомастами",     "двохстах"     ],
+    [ "триста",    "трьохсот",    "трьомстам",    "триста",    "трьомастами",    "трьохстах"    ],
+    [ "чотириста", "чотирьохсот", "чотирьомстам", "чотириста", "чотирмастами",   "чотирьохстах" ],
+    [ "пʼятсот",   "пʼятисот",    "пʼятистам",    "пʼятсот",   "пʼятьмастами",   "пʼятистах"    ],
+    [ "шістсот",   "шестисот",    "шестистам",    "шістсот",   "шістьмастами",   "шестистах"    ],
+    [ "сімсот",    "семисот",     "семистам",     "сімсот",    "сімомастами",    "семистах"     ],
+    [ "вісімсот",  "восьмисот",   "восьмистам",   "вісімсот",  "восьмистами",    "восьмистах"   ],
+    [ "девʼятсот", "девʼятисот",  "девʼятистам",  "девʼятсот", "девʼятьмастами", "девʼятистах"  ],
+];
+
+const HUNDRED_BASE: &str = "сот";
+
+#[rustfmt::skip]
+const THOUSAND_FLEXIONS: [[&str; 6]; 2] = [
+    [ "а", "і", "і",  "у", "ею",  "і"  ],
+    [ "і", "",  "ам", "і", "ами", "ах" ],
+];
+
+// Number names by "rule n-1" from https://uk.wikipedia.org/wiki/Іменні_назви_степенів_тисячі
+const MEGA_BASES: [&str; 21] = [
+    "тисяч",
+    "мільйон",
+    "мільярд",
+    "трильйон",
+    "квадрильйон",
+    "квінтильйон",
+    "секстильйон",
+    "септильйон",
+    "октильйон",
+    "нонильйон",
+    "децильйон",
+    "ундецильйон",
+    "додецильйон",
+    "тредецильйон",
+    "кваттуордецильйон",
+    "квіндецильйон",
+    "седецильйон",
+    "септдецильйон",
+    "дуодевігінтильйон",
+    "ундевігінтильйон",
+    "вігінтильйон",
+];
+
+#[rustfmt::skip]
```

Comment:
why so many rustfmt::skip? I'd rather keep the default, recommended formatting for most objects

Reply:
Tables. Without rustfmt::skip, the table layout is broken, and looking for the correct declination will be quite painful. I'll review all rustfmt::skip and, maybe, remove some, but I'm pretty sure I need several of them. Compare
```
    [ "сто",       "ста",         "ста",          "сто",       "ста",            "ста"          ],
    [ "двісті",    "двохсот",     "двомстам",     "двісті",    "двомастами",     "двохстах"     ],
```
to
```
    ["сто", "ста", "ста", "сто", "ста", "ста"],
    ["двісті", "двохсот", "двомстам", "двісті", "двомастами", "двохстах"],
```
The alternatives, like is adding declinations in comments, are much worse:
```
    [ /*nom*/ "сто", /*gen*/ "ста", /*dat*/ "ста", /*acc*/ "сто", /*ins*/ "ста", /*loc*/ "ста" ],
    [ /*nom*/ "двісті",  /*gen*/ "двохсот", /*dat*/ "двомстам", /*acc*/ "двісті", /*ins*/"двомастами", /*loc*/ "двохстах"     ],
```
You may check table formatting in Python source code: https://github.com/savoirfairelinux/num2words/blob/master/num2words/lang_UK.py

Reply:
LGTM like that, the table layout is better as is I agree, it was mostly with the over-abundance of those, which you've tackled, thanks!